client/shipping-label/views/index.js

```diff
@@ -49,15 +52,49 @@ class ShippingLabelRootView extends Component {
 		this.setState( { showTooltips } );
 	}
 
+	renderPaymentInfo() {
+		const paymentMethod = this.props.shippingLabel.paymentMethod;
+
+		if ( paymentMethod ) {
+			return (
+				<Notice isCompact={ true } showDismiss={ false } className="wcc-metabox-label-payment inline">
+					<p dangerouslySetInnerHTML={ { __html: sprintf(
```

Comment:
No need to use `dangerouslySetInnerHTML` here, you can just print the `<strong>${digits}</strong>` outside of the `sprintf` string.